src/pal/tools/cppmunge/CMakeLists.txt

```diff
@@ -10,6 +10,10 @@ project(cppmunge)
 
 add_definitions(-D_BSD_SOURCE) # Needed to get realpath part of CRT for c99 compat.
 
+if(CLR_CMAKE_PLATFORM_LINUX)
+  add_definitions(-D__LINUX__=1)
```

Comment:
I'm not too sure about that. Anyway, I just followed the general methodology from the base CMakeLists.txt in the repo:

https://github.com/dotnet/coreclr/blob/master/CMakeLists.txt#L350-L354